Description: use RANDFILE
 If unconfigured, openssl leaves a /root/.rnd file around after running the
 stenokeys command in the stenographer-common postinst script. This patch
 makes openssl use /dev/urandom as a seed, and also redirects any write to
 this device, avoiding creation of a file.
Author: Sascha Steinbiss <satta@debian.org>
--- a/stenokeys.sh
+++ b/stenokeys.sh
@@ -26,6 +26,7 @@
 USR=$1
 GRP=$2
 CONFIG=$(mktemp -t stenossl.XXXXXXXXXXXX)
+export RANDFILE=/dev/urandom
 
 function sslconfig_common {
 cat <<EOF
